azext_iot/tests/test_monitor_parsers_unit.py

```diff
@@ -100,6 +114,7 @@ def test_parse_message_should_succeed(
         payload,
         properties,
         app_properties,
+        module_id,
     ):
         # setup
         properties = MessageProperties(
```

Comment:
lets also add an assert on the module name being what we passed in

Reply:
Should be able to add a similar snippet to whats on 143:
![image](https://user-images.githubusercontent.com/42591144/83175700-2b1ab100-a0d1-11ea-9652-b9a4d8966ca3.png)

```
if module_id:
    assert parsed_msg["event"]["annotations"][module_identifier] == module_id
else:
    assert not parsed_msg["event"]["annotations"].get(module_identifier)
```